src/deployment-guides/tko-saas-services.md

```diff
@@ -10,18 +10,21 @@ The following VMware SaaS services provide additional Kubernetes lifecycle manag
 
 Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds. It provides operators with a single control point to give developers the independence they need to drive business forward, while enabling consistent management and operations across environments for increased security and governance.
 
-We recommended attaching the shared services and workload clusters to Tanzu Mission Control as it enables you to manage your global portfolio of Kubernetes clusters from a centralized a centralized administrative interface.
+It also allows you to deploy Tanzu Kubernetes clusters directly from TMC and install user-managed Packages leveraging the [TMC Catalog](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-EF35646D-8762-41F1-95E5-D2F35ED71BA1.html) feature. 
```

Comment:
You can also deploy Tanzu Kubernetes clusters directly from Tanzu Mission Control and install user-managed packages from the Tanzu Mission Control Catalog page. For more information, see [Managing Packages in Your Cluster] (https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-EF35646D-8762-41F1-95E5-D2F35ED71BA1.html)